WebRoot/js/zimbraMail/abook/view/ZmGroupView.js

```diff
@@ -554,7 +554,7 @@ function(x, y, width, height) {
 	if(this._addNewField){
 		Dwt.setSize(this._addNewField, Dwt.DEFAULT, 100);
 	}
-	var memberListHeight = height - 50;
+	var memberListHeight = height - 150;
```

Comment:
can we add a comment what determines the subtract value . Could be helpful for someone looking this in future.